### ISSUE TITLE
judoscale-sidekiq causing deprecation warnings in sidekiq 7.2.1

### ISSUE THREAD

**Issue by WillRogers727** [judoscale/judoscale-ruby#195]
Since updating sidekiq to v7.2.1 we've been getting a lot of deprecation warnings like this:

```
/app\/vendor\/bundle\/ruby\/3.2.0\/gems\/judoscale-sidekiq-1.5.3\/lib\/judoscale\/sidekiq\/metrics_collector.rb:24: warning: Direct access to `Sidekiq::Work` attributes is deprecated, please use `#payload`, `#queue`, `#run_at` or `#job` instead/
```

It appears to be caused by accessing `work["payload"]` in `lib/judoscale/sidekiq/metrics_collector.rb:24`

Relevant sidekiq commit: https://github.com/sidekiq/sidekiq/commit/da03008cd536682d8a3fe9bcb1ac727b490eda27

It should be possible to solve by changing the way of accessing work to `work.payload` 

Let me know if any other info is needed. Thanks!

**Comment by adamlogic** [comment]
Took me a while to repro this because I wasn't seeing the warning output. This article helped a lot: https://www.fastruby.io/blog/exploring-ruby-warnings.html

Starting my process with `RUBYOPT="-W2"` did the trick.

Anyway, I'm pushing a fix now. Thanks for the report!

**Comment by adamlogic** [comment]
Released in v1.5.4

**Comment by WillRogers727** [comment]
> Released in v1.5.4

Thanks for the quick release! :D
